pai-management/Dockerfile

```diff
@@ -60,13 +60,13 @@ RUN wget https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-c
 RUN cp docker-17.06.2-ce.tgz /usr/local
 RUN tar xzvf /usr/local/docker-17.06.2-ce.tgz
 
-COPY container-setup.sh /
-
 COPY k8sPaiLibrary/maintaintool/kubectl-install.sh /kubectl-install.sh
 RUN /bin/bash kubectl-install.sh
 
 RUN echo y | pip uninstall requests
```

Comment:
Can you modify these 4 ```RUN``` into one?

Reply:
Hi @ydye ,
It will need some trick.
There is a pipe input, the fellow doesn't work:
```
RUN echo y | pip uninstall requests && \
  echo y | pip install requests
```

Reply:
Done